### PR TITLE
Monster Shop Group Project : Navigation US #3

### PR DESCRIPTION
**Pull request by aperezsantos**
User Story 3 goes through setting up the navigation for a default user (user). From the navigation bar, a user is able to link to a welcome page, merchant and item index pages, a cart page, and their profile page, are able to see a message indicating that they are logged in, and can log out.

It is important to note that a user will not see login or register links at this point.

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
   def cart
     @cart ||= Cart.new(session[:cart] ||= Hash.new(0))
   end
 
+  def current_user
+    @current_user ||= User.find(session[:user_id]) if session[:user_id]
+  end
+
+  def current_user?
+    current_user && current_user.user?
```
Comment (takeller) [inline app/controllers/application_controller.rb:21]: This is confusing. Should we rename a registered user as default?

**Review by takeller — COMMENTED** [review]
Can you merge the nav into your user story locally so we can avoid some of the merge conflicts?

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (aperezsantos) [reply app/controllers/application_controller.rb:21]: Based on the user stories, a registered user is the default user. In our setup, we named a default user role just user, and it's a bit confusing for sure. I would agree that we should come up with a definitive term for a registered (default) user.

Comment (madhalle) [reply app/controllers/application_controller.rb:21]: rename to registered_user to keep things clear

**Comment by madhalle** [comment]
Looks good! Need to move logged in chunk down.

**Pull request merged**